ignite/metrics/confusion_matrix.py

```diff
@@ -13,6 +13,10 @@
 class ConfusionMatrix(Metric):
     """Calculates confusion matrix for multi-class data.
 
+    The confusion matrix is formatted such that columns are predictions and rows are targets.
```

Comment:
Let's move this details below after the line L67 and use `Note:` : 

```python
Note:
    The confusion matrix is formatted such that columns are predictions and rows are targets.
    For example, if you were to plot the matrix, you could correctly assign to the horizontal axis
    the label "predicted values" and to the vertical axis the label "actual values".
```

Reply:
Sounds great! @vfdev-5 , should I do the edit? If yes, how? :)

Reply:
Yes, let's edit the file locally as suggested. Next, commit and push: `git commit -a -m "Updated docstring" && git push origin fix-docs-2195`

Reply:
Done. Let me know if that's fine.

Reply:
Yes, IMO looks good now! Thanks for the PR !

Reply:
Thank you for the guidance!